module-4/backend/week_one.md

```diff
@@ -5,16 +5,53 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
+- Learning the basic syntax of JS and the major differences between it and Ruby was very helpful. Learning that JS requires explicit returns and that to make a copy of an array, you need to use Array.prototype.slice() was really useful.
+
 2. What are some tools to help debug JavaScript code?
+- Pryjs, debugger and the console.
+
 3. What are some tools you need in order to unit test your JavaScript?
+- chai, mocha
+
 4. What is the syntax for invoking a function?
+```
+function addTwo (a, b) {
+  return (a + b)
+  }
+  
+  addTwo (3, 5)
```

Comment:
Great! When actually invoking, you should remove the space between the signature and arguments. i.e., `addTwo(3, 5)`